### PR TITLE
Refactors and restyles executions page

### PR DESCRIPTION
**Pull request by sorsaffari**
- ~~uses a table to display list of executions and so removes `ExecutionCard.vue`~~
- adds a TopBar to the main layout.
- improves overall styling
- brings in font-awesome as a dependency to be used as the icon library
- implements `parseDate` as a global filter so that is can be reused across components
- ~~implements `coptyToClipboard` as a mixin so that is can be reused across components~~
- uses [copy-to-clipboard](https://github.com/sudodoki/copy-to-clipboard)
- Improves UX on Executions page:
    - ~~hides~~ disables certain actions when execution is complete
    -  asks for confirmation on sensitive actions (stop or delete)
    -  adds tooltips to card details
    -  shows a loading spin while data is being fetched

### CODE REVIEW

File: service/dashboard/src/views/executions/Executions.vue [hunk 1]
```diff
@@ -1,129 +1,162 @@
...
+      label="Completed">
+      <template slot-scope="scope">
+        {{ scope.row.executionCompletedAt | parseDate }}
+      </template>
+    </el-table-column>
+    <el-table-column
+      prop="actions"
+      label="Actions"
+      width="100">
+      <template slot-scope="scope">
```
Comment (marco-scoppetta) [inline service/dashboard/src/views/executions/Executions.vue]: all these templates are the reason why I rather have:
```
<execution-card
   v-for="exec in executions"
   :key="exec.id"
   :execution="exec"
/>
```

in there you get the single execution and you can do everything within a template without all the additional logic

File: service/dashboard/src/util/mixins.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const copyToClipboard = {
```
Comment (marco-scoppetta) [inline service/dashboard/src/util/mixins.js]: this should just be an helper function in `Executions.vue` given that is not used by any other component

File: service/dashboard/src/views/executions/Executions.vue [hunk 3]
```diff
@@ -1,129 +1,162 @@
...
+            .catch((e) => {
+              console.log(e);
+              this.$message({ showClose: true, message: 'Deleting the execution failed. Check the console logs to find out why.', type: 'error' });
+            } );
+        })
+        .catch(_ => {});
+    },
+    stopExecution(execution) {
+      this.$confirm("Are you sure you want to stop this execution?")
+        .then(_ => {
```
Comment (marco-scoppetta) [inline service/dashboard/src/views/executions/Executions.vue]: what's `_`

File: service/dashboard/src/views/executions/Executions.vue [hunk 4]
```diff
@@ -1,129 +1,162 @@
...
 
-<style scoped>
-.el-container {
-  padding: 0 30px;
-  background-color: #f4f3ef;
-}
+<style lang="scss" scroped>
+  td:last-child, th:last-child { text-align: right; }
+  .el-table__body-wrapper::-webkit-scrollbar { display: none; }
+  [class^="status-"] {
```
Comment (marco-scoppetta) [inline service/dashboard/src/views/executions/Executions.vue]: regex not needed

File: service/dashboard/src/views/executions/Executions.vue (hunk 3 above)
Comment (sorsaffari) [reply service/dashboard/src/views/executions/Executions.vue]: did a quick search and found out:
> `_` explicitly states that the function will be passed one argument, but that you don't care about it.
more about it here: https://stackoverflow.com/a/41085908/10600803

I'm switching back to `()` as I found `_` to cause more confusion for future team members.

File: service/dashboard/src/views/executions/Executions.vue (hunk 4 above)
Comment (sorsaffari) [reply service/dashboard/src/views/executions/Executions.vue]: 👍

File: service/dashboard/src/util/mixins.js (hunk 2 above)
Comment (sorsaffari) [reply service/dashboard/src/util/mixins.js]: decided to use [copy-to-clipboard](https://github.com/sudodoki/copy-to-clipboard) instead

File: service/dashboard/src/views/executions/Executions.vue (hunk 1 above)
Comment (sorsaffari) [reply service/dashboard/src/views/executions/Executions.vue]: switched back to using cards

**Pull request closed without merging**